src/locator/index.ts

```diff
@@ -62,6 +62,10 @@ function computeDimension(topLeft: Point, topRight: Point, bottomLeft: Point, ma
     sum(countBlackWhiteRun(topRight, topLeft, matrix, 5)) / 7
   ) / 4;
 
+  if (moduleSize === 0) {
```

Comment:
Unless 0 is a magic number being returned by something, it'd probably be better to switch this to something like `if (moduleSize < 1) {` or whatever the smallest reasonable module size is to avoid float rounding issues.

Reply:
So at least with the test case this was triggering because `countBlackWhiteRun` was returning `[0,0,-1,0,0]` which basically means the finder patterns were too small to extract distances from. We could try throwing deeper in the stack but it felt reasonable to just check for an explicit 0 here. Thoughts?